casper/src/main/scala/coop/rchain/casper/util/comm/DeployRuntime.scala

```diff
@@ -110,6 +110,8 @@ object DeployRuntime {
       )
     )
 
+  def status[F[_]: Sync: DeployService]: F[Unit] = gracefulExit(DeployService[F].status)
```

Comment:
It is fine to have it in command line although I don't think people use that now

Reply:
We should be consistent with API interface in any supported protocol. Eventually we should have only one API implementation with many protocols preferably just generated from the API interface.